src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Helper/Base64DecoderHelper.cs

```diff
@@ -0,0 +1,1446 @@
+                    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+                    -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
+                ];
+
+            public ReadOnlySpan<uint> VbmiLookup0 =>
+                [
+                    0x80808080, 0x80808080, 0x80808080, 0x80808080,
+                    0x80808080, 0x80808080, 0x80808080, 0x80808080,
+                    0x80808080, 0x80808080, 0x3e808080, 0x3f808080,
+                    0x37363534, 0x3b3a3938, 0x80803d3c, 0x80808080
+                ];
+
+            public ReadOnlySpan<uint> VbmiLookup1 =>
+                [
+                    0x02010080, 0x06050403, 0x0a090807, 0x0e0d0c0b,
+                    0x1211100f, 0x16151413, 0x80191817, 0x80808080,
+                    0x1c1b1a80, 0x201f1e1d, 0x24232221, 0x28272625,
+                    0x2c2b2a29, 0x302f2e2d, 0x80333231, 0x80808080
+                ];
+
+            public ReadOnlySpan<sbyte> Avx2LutHigh =>
+                [
+                    0x10, 0x10, 0x01, 0x02,
+                    0x04, 0x08, 0x04, 0x08,
+                    0x10, 0x10, 0x10, 0x10,
+                    0x10, 0x10, 0x10, 0x10,
+                    0x10, 0x10, 0x01, 0x02,
+                    0x04, 0x08, 0x04, 0x08,
+                    0x10, 0x10, 0x10, 0x10,
+                    0x10, 0x10, 0x10, 0x10
+                ];
+
+            public ReadOnlySpan<sbyte> Avx2LutLow =>
+                [
+                    0x15, 0x11, 0x11, 0x11,
+                    0x11, 0x11, 0x11, 0x11,
+                    0x11, 0x11, 0x13, 0x1A,
+                    0x1B, 0x1B, 0x1B, 0x1A,
+                    0x15, 0x11, 0x11, 0x11,
+                    0x11, 0x11, 0x11, 0x11,
+                    0x11, 0x11, 0x13, 0x1A,
+                    0x1B, 0x1B, 0x1B, 0x1A
+                ];
+
+            public ReadOnlySpan<sbyte> Avx2LutShift =>
+                [
+                    0, 16, 19, 4,
+                    -65, -65, -71, -71,
+                    0, 0, 0, 0,
+                    0, 0, 0, 0,
+                    0, 16, 19, 4,
+                    -65, -65, -71, -71,
+                    0, 0, 0, 0,
+                    0, 0, 0, 0
+                ];
+
+            public byte MaskSlashOrUnderscore => (byte)'/';
+
+            public ReadOnlySpan<int> Vector128LutHigh => [0x02011010, 0x08040804, 0x10101010, 0x10101010];
+
+            public ReadOnlySpan<int> Vector128LutLow => [0x11111115, 0x11111111, 0x1A131111, 0x1A1B1B1B];
+
+            public ReadOnlySpan<uint> Vector128LutShift => [0x04131000, 0xb9b9bfbf, 0x00000000, 0x00000000];
+
+            public ReadOnlySpan<uint> AdvSimdLutOne3 => [0xFFFFFFFF, 0xFFFFFFFF, 0x3EFFFFFF, 0x3FFFFFFF];
+
+            public uint AdvSimdLutTwo3Uint1 => 0x1B1AFFFF;
+
+            public int GetMaxDecodedLength(int utf8Length) =>
+#if NETCOREAPP
+                Base64.GetMaxDecodedFromUtf8Length(utf8Length);
+#else
+                0;
```

Comment:
Why is this 0?

Reply:
Well, this source will not called for netsandard,  though I think now after splitting the Base64 public vs internal sources I don't need to do this.